docs/build/configure/_index.md

```diff
@@ -381,11 +381,42 @@ Switch to **Raw JSON** mode to configure webhooks as follows:
 
 ## Fragments
 
-Fragments are a way of sharing and managing identical {{< glossary_tooltip term_id="resource" text="resource" >}} configuration files across multiple machines.
-For example, if you have multiple machines with the same hardware, wired the same way, you can create and share a fragment and add it to any number of machines.
-When changes are made to the fragment, those changes are automatically carried to all machines that include the fragment in their config.
+You can use fragments to share identical {{< glossary_tooltip term_id="resource" text="resource" >}} configuration files across multiple machines in your fleet.
```

Comment:
We went back and forth on this phrasing when @JessamyT wrote the configure a fleet page. I don't like the word "identical" here. Because it can be a partial configuration that is shared. And then this can be too easily misconstrued as "it's not possible to use fragments unless all my machines are exactly identical"

Reply:
I see that the existing sentence still contains the word "identical"--perhaps I should've changed that in https://github.com/viamrobotics/docs/pull/2466 in addition to changing the wording on the other page--oops! Agree it should be updated here. I think the next sentence with the example of one specific component does help illustrate the point that it could be for just one or more components etc. among many.

Reply:
Updating to just `similar` in both places, good points all!